code-challenges.test.js

```diff
@@ -22,12 +22,38 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
-
+// a describe method that lists the name of the function OR naming of the particular test.
+describe("evenlyDivisibleByThreeOrNot", () => {
+
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("logs decides if the number is evenly divisble by three or not", () => {
+  
+      //an expect method, nested within the test block, calling on the evenlyDivisibleByThreeOrNot() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(evenlyDivisibleByThreeOrNot(15)).toEqual("15 is divisible by three")
+      expect(evenlyDivisibleByThreeOrNot(0)).toEqual("0 is divisible by three")
+      expect(evenlyDivisibleByThreeOrNot(-7)).toEqual("-7 is not divisible by three")
+    })
+  })
```

Comment:
Your test looks great. But the reason for the test variables is to use them in your code.